api/proquest/importer.py

```diff
@@ -230,7 +231,7 @@ def __init__(
         super(ProQuestOPDS2Importer, self).__init__(
             db,
             collection,
-            parser,
+            parser if parser else RWPMManifestParser(OPDS2FeedParserFactory()),
```

Comment:
It would be good to have a test that ensures `super` called with correct value when parser=None.